### DISCUSSION TITLE
VSCode Extension: Requested codespace not found

### DISCUSSION THREAD

**Discussion by tamirkamara** [community/community#5300] [Codespaces]
Hi,
I've created a codespace for several repos, both my own and ones owned by an org, and the web experience works great. However the VSCode experience doesn't work at all for me and I don't understand why... I tried to open codespaces in VSCode through the web interface and get the message below, I also don't see any codespaces listed inside the VSCode extension side bar.
I tried to re-install the extension but that didn't change anything.
Any help will be appreciated. 

![image](https://user-images.githubusercontent.com/26870601/130907801-8e9b4553-801e-49a9-8f1a-9c39ed6e25b3.png)

**Comment by tanmayeekamath** [comment]
Hey @tamirkamara, thanks for reaching out. Can you please create a support ticket here: https://support.github.com/
Our team can help investigate. Thanks for your patience :)

**Reply by fitzgeraldsteele** [reply]
Also having this issue. Will also open a ticket
